src/osd/PrimaryLogPG.cc

```diff
@@ -1858,6 +1858,11 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
     osd->reply_op_error(op, -ENAMETOOLONG);
     return;
   }
+  if (m->get_hobj().oid.name.empty()) {
+    dout(4) << "do_op empty oid name is not allowed" << dendl;
```

Comment:
should we include a cluster log here so that we catch it if rgw/rbd/cephfs does this during testing?